packages/terra-functional-testing/lib/logger/logger.js

```diff
@@ -0,0 +1,27 @@
+class Logger {
+  constructor(options) {
+    const { prefix } = options;
+
+    this.prefix = prefix;
+  }
+
+  /**
+   * Formats the text.
+   * @param {string} text - The text to format.
+   * @returns {string} - A formatted string.
+   */
+  format(text) {
+    return `[terra-functional-testing:${this.prefix}] ${text}`;
```

Comment:
colored logs? 😄

Reply:
Starting small with this logger. When I run into a situation that needs to log different tiers of output I'll likely add in colored logs. The [WebDriver logger](https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-logger) has differences between their INFO, DEBUG, ERROR logs. Similar to our existing logger. For this PR I don't intend to add colored logs. I think it would be good to have when we discuss this in more detail:

https://github.com/cerner/terra-toolkit/pull/412#discussion_r433853473

Reply:
Should we add this to the list of v6 list, whether we do it or not, so we don't forget?

Reply:
I created a new issue to track the improvements to the logger. Feel free to add/edit the issue with more information/suggetions/requests:

https://github.com/cerner/terra-toolkit/issues/414